rpm/polkit.spec

```diff
@@ -5,50 +5,79 @@ Release:    1
 License:    LGPLv2+
 URL:        http://www.freedesktop.org/wiki/Software/PolicyKit
 Source0:    %{name}-%{version}.tar.gz
-Source1:    dbus-org.freedesktop.PolicyKit1.service
-Patch0:     patches/0.108/build-Fix-.gir-generation-for-parallel-make.patch
-Patch1:     patches/0.110/07_set-XAUTHORITY-environment-variable-if-unset.patch
-Patch2:     patches/0.110/04_get_cwd.patch
-Patch3:     patches/0.111/09_pam_environment.patch
-Patch4:     patches/0.112/00git_type_registration.patch
-Patch5:     patches/0.112/08_deprecate_racy_APIs.patch
-Patch6:     patches/0.112/cve-2013-4288.patch
-Patch7:     patches/0.113/Port-internals-non-deprecated-PolkitProcess-API-wher.patch
-Patch8:     patches/0.113/pkexec-Work-around-systemd-injecting-broken-XDG_RUNT.patch
-Patch9:     patches/0.113/03_PolkitAgentSession-fix-race-between-child-and-io-wat.patch
-Patch10:    patches/0.113/polkitd-Fix-problem-with-removing-non-existent-sourc.patch
-Patch11:    patches/0.113/PolkitSystemBusName-Add-public-API-to-retrieve-Unix-.patch
-Patch12:    patches/0.113/Fixed-compilation-problem-in-the-backend.patch
-Patch13:    patches/0.113/Don-t-discard-error-data-returned-by-polkit_system_b.patch
-Patch14:    patches/0.113/sessionmonitor-systemd-Deduplicate-code-paths.patch
-Patch15:    patches/0.113/sessionmonitor-systemd-prepare-for-D-Bus-user-bus-mo.patch
-Patch16:    patches/0.113/Refuse-duplicate-user-arguments-to-pkexec.patch
-Patch17:    patches/0.113/00git_fix_memleak.patch
-Patch18:    patches/0.113/00git_invalid_object_paths.patch
-Patch19:    patches/0.113/sessionmonitor-systemd-Use-sd_uid_get_state-to-check.patch
-Patch20:    patches/0.113/Fix-a-possible-NULL-dereference.patch
-Patch21:    patches/0.113/Fix-duplicate-GError-use-when-uid-is-missing.patch
-Patch22:    patches/0.113/Fix-a-crash-when-two-authentication-requests-are-in-.patch
-Patch23:    patches/0.113/CVE-2015-4625-Use-unpredictable-cookie-values-keep-t.patch
-Patch24:    patches/0.113/CVE-2015-4625-Bind-use-of-cookies-to-specific-uids.patch
-Patch25:    patches/0.113/docs-Update-for-changes-to-uid-binding-Authenticatio.patch
-Patch26:    patches/0.113/Fix-a-per-authorization-memory-leak.patch
-Patch27:    patches/0.113/Fix-a-memory-leak-when-registering-an-authentication.patch
-Patch28:    patches/0.113/CVE-2015-3255-Fix-GHashTable-usage.patch
-Patch29:    patches/0.113/Fix-use-after-free-in-polkitagentsession.c.patch
-Patch30:    patches/0.113/README-Note-to-send-security-reports-via-DBus-s-mech.patch
-Patch31:    patches/master/Fix-multi-line-pam-text-info.patch
-Patch32:    patches/CVE-2018-1116.patch
-Patch33:    patches/0001-dbus-Use-systemd-service.patch
-Patch34:    patches/0002-build-Disable-gtk-doc-support.patch
-Patch35:    patches/0003-Support-for-annotation-identity-group-check.patch
-Patch36:    patches/0004-Fix-build-with-systemd-v233.patch
+Source1:    polkit.service
+Source2:    genpatchlist.sh
+Source3:    rpm/patches/debian/series
```

Comment:
I think you are not supposed to include `rpm/` directory itself in source file paths.